docs/troubleshooting.rst

```diff
@@ -193,3 +193,16 @@ Now build again::
     tutor images build
 
 All build commands should now make use of the newly configured builder. To later revert to the default builder, run ``docker buildx use default``. 
+
+fatal: the remote end hung up unexpectedly / fatal: early EOF / fatal: index-pack failed when running `tutor images build ...`
+------------------------------------------------------------------------------------------------------------------------------
+
+This issue can occur due to problems with your network while cloning edx-platform which is a fairly large repository.
+
+First, try to run the same command once again to see if it works as your network connection can sometimes drop during the build process.
+
+If that does not work, you can follow the tutorial above for `High resource consumption <#high-resource-consumption-on-tutor-images-build-by-docker>`_ to limit the number of concurrent build steps so that your network is not being shared between multiple layers at once.
```

Comment:
Let's use a ref for internal linking. (see instances of `:ref:`)

Reply:
Replaced with `:ref:`